pom.xml

```diff
@@ -304,9 +304,14 @@
       <version>2.11.0</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpmime</artifactId>
-      <version>4.5.14</version>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+      <version>5.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents.core5</groupId>
+      <artifactId>httpcore5</artifactId>
+      <version>5.1</version>
```

Comment:
Actually one minor change - can we go straight to the latest available? Also a quick double check whether we require httpcore5 separately.

Reply:
version is updated...we are using some imports from httpcore which are not available in httpclient5 so we cant remove that